test/source/tests/compose.ts

```diff
@@ -2296,30 +2288,28 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
           // check if flowcrypt keyserver results are priotized than keyserver.pgp.com results
           bcc: 'test.flowcrypt.pubkeyserver.priority@gmail.com',
         };
-        t.mockApi!.configProvider = new ConfigurationProvider({
-          attester: {
-            pubkeyLookup: {
-              [recipients.to]: {
-                pubkey: somePubkey,
-              },
-              [recipients.bcc]: {
-                pubkey: somePubkey,
-              },
+        t.mockApi!.configProvider!.config!.attester = {
```

Comment:
I propose merging this pull request after #5076.
#5076 is expected to be ready today, and once it is merged, we can revert the changes made in this pull request (https://github.com/FlowCrypt/flowcrypt-browser/pull/5087/commits/37c2fa400800cc1a318cc1dde19432d2919c1026) and close it.

Reply:
Sounds reasonable, let's wait for finishing https://github.com/FlowCrypt/flowcrypt-browser/pull/5083, as it part of our bigger test refactoring, then it probably should fix this issue too.